tests/behat/features/capabilities/groups/join/groups-join-request-manage.feature

```diff
@@ -0,0 +1,84 @@
+@api
+Feature: Requests to join for a group can be managed
+
+  Background:
+    Given I enable the module social_group_flexible_group
+    And I enable the module social_group_request
+    And groups with non-anonymous owner:
+      | label      | field_group_description | type           | langcode | field_flexible_group_visibility | field_group_allowed_join_method |
+      | Test group | Public visibility       | flexible_group | en       | public                          | request                         |
+    And users:
+      | name           | mail                | status | roles    |
+      | Pending Member | pending@example.com | 1      | verified |
+    And group membership requests:
+      | group      | user           |
+      | Test group | Pending Member |
+
+  Scenario: As a group manager I can approve a membership request
+    Given I am logged in as a user with the "verified" role
+    # Todo: Fix the role declaration.
+    And I am a member of "Test group" with the "flexible_group-group_manager" role
+
+    When I am viewing the "membership-requests" page of group "Test group"
+    And I press "Approve Membership"
+    And I press "Yes"
+
+    Then I should see the success message "Membership request approved"
+    And I should see "No membership requests available."
+
+  Scenario: As a group manager I can reject a membership request
+    Given I am logged in as a user with the "verified" role
+    # Todo: Fix the role declaration.
+    And I am a member of "Test group" with the "flexible_group-group_manager" role
+
+    When I am viewing the "membership-requests" page of group "Test group"
+    And I press "Toggle Dropdown"
+    And I click "Reject Membership"
+    And I press "Yes"
+
+    Then I should see the success message "Membership request rejected"
+    And I should see "No membership requests available."
```

Comment:
In my opinion, checking that there are no memberships requests anymore is not enough, I would add a validation to certify that the manager rejection was executed, by, for example, check if the rejected member is NOT listed under members tab.